File: src/main/java/dev/su5ed/koremods/modlaunch/Koremods.java
package dev.su5ed.koremods.modlaunch;

import net.minecraftforge.fml.common.Mod;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

@Mod("koremods")
public class Koremods { // TODO
    private static final Logger LOGGER = LogManager.getLogger();
    
    public Koremods() {
        LOGGER.info("Mod Constructed");
    }
}

File: src/main/kotlin/dev/su5ed/koremods/modlaunch/Koremods.kt
package dev.su5ed.koremods.modlaunch

import net.minecraftforge.fml.common.Mod
import org.apache.logging.log4j.LogManager
import org.apache.logging.log4j.Logger

@Mod("koremods")
class Koremods {
    private val logger: Logger = LogManager.getLogger()
    
    init {
        logger.info("Mod Constructed")
    }
}
